src/cryptoadvance/specter/devices/hwi/jade.py

```diff
@@ -163,12 +164,16 @@ def __init__(
                     "Use JadeAPI.set_[seed|mnemonic] to set simulator wallet"
                 )
         else:
-            if uninitialized and not HAS_NETWORKING:
-                # Wallet not initialised/unlocked nor do we have networking dependencies
-                # User must use 'Recovery Phrase Login' or 'QR Unlock' feature to access wallet
-                raise DeviceNotReadyError(
-                    'Use "Recovery Phrase Login" or "QR PIN Unlock" feature on Jade hw to access wallet'
-                )
+            if uninitialized:
```

Comment:
The below checks seem the wrong way around ... I mean if 'unlock' is false (and you want that to return normally without error), then there's no need to raise an exception if networking is missing.

Reply:
Makes sense, will change that.